modules/light-clients/07-tendermint/client_state.go

```diff
@@ -218,6 +218,15 @@ func (cs ClientState) VerifyMembership(
 	path exported.Path,
 	value []byte,
 ) error {
+	// check if there is a proof passed in
```

Comment:
mega nit, but could we remove this comment maybe? I don't think it adds much value